### ISSUE TITLE
Include a guide on how to install this

### ISSUE THREAD

**Issue by BenNewsome** [alfredodeza/pytest.vim#46]
Do I just copy the pytest.vim file to my .vim/ftplugin or is there something I am missing.

I also have vundle installed which appears to require filetype off.

As this appears to require filetype on, does this mean this is incompatible?

**Comment by alfredodeza** [comment]
An installation guide would fail pretty quickly to address what you describe in this issue. I am not sure how `vundle` works. For example, I use `pathogen` and that only needs the full checkout of this repository in `.vim/bundle/`

You are correct that this is a `filetype` plugin and it is required to have `filetype=on` to make it work.

Filetype plugins require to be in the right directory for the file type they support. In this case Python. You can see that in the path where the plugin exists: `ftplugin/python/pytest.vim`

I don't mind accepting a PR that addresses the "how to install this with vundle". Hopefully my pointers can help you out.

**Comment by BenNewsome** [comment]
I have got it to work. Thanks.
I did not know it had to be in .vim/bundle/
Vundle is a bundle manager from the looks of it, where you just have to type the github username/project title.
So to install this in vundle I had to add to my vimrc in the correct location:
`Plugin alfredodeza/pytest.vim
`
And then do` :PluginInstall`
I am pretty new to plugins in vim and so just saying something like install by cloning into .vim/bundle or use a vim package manager would go a long way to helping other newbies.

Thanks for the plugin and the help. If you don't think this is needed feel free to close the issue.
